config.go

```diff
@@ -72,7 +72,7 @@ func readConfigFile(config interface{}, fileName string) error {
 	return nil
 }
 
-func (c menderConfig) GetUpdaterConfig() httpsClientConfig {
+func (c menderConfig) GetHttpConfig() httpsClientConfig {
```

Comment:
In fact we are returning `httpsClientConfig` here, some maybe at least `GetHttpsClientConfig()`?